doc/sphinx/source/community/release_strategy.rst

```diff
@@ -79,6 +67,20 @@ Upcoming releases
 Past releases
 ^^^^^^^^^^^^^
 
+- 2.6.0 (Release Manager: `Saskia Loosveldt Tomas`_)
+
++------------+------------+---------------------------------------------------------------------------------------------+------------------------------------+
+|  Planned   |    Done    |                                            Event                                            |             Changelog              |
++============+============+=============================================================================================+====================================+
+| 2022-06-06 |            |                                  ESMValCore Feature Freeze                                  |                                    |
++------------+------------+---------------------------------------------------------------------------------------------+------------------------------------+
+| 2022-06-13 | 2022-06-15 | `ESMValCore Release 2.6.0 <https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.6.0>`_ | :ref:`esmvalcore:changelog-v2-6-0` |
```

Comment:
I wish it would have been in June, but both release dates have been in July :sweat_smile:

Reply:
shush, nobody know :grin:

Reply:
haha, i was already surprised that you did the release 2 days early 😄